src/main.c

```diff
@@ -286,18 +204,12 @@ static int server_find_gid(struct ip2gid_req_ipv4 *req,
 			   struct ip2gid_resp_gid *resp,
 			   uint32_t *resp_len)
 {
-	char gid_str[INET6_ADDRSTRLEN] = {};
-	char ip_str[INET_ADDRSTRLEN] = {};
 	struct nl_addr *nl_addr;
 	struct rtnl_link *link;
 	struct ifaddrs* ifaddr;
 	struct ifaddrs* ifa;
 	int err;
 
-	ip2gid_inet_ntop(IP2GID_LOG_INFO, AF_INET, (const void *)(&req->ipv4),
-			 ip_str, sizeof(ip_str));
-	ip2gid_log(IP2GID_LOG_INFO,
```

Comment:
why are you removing this?
This is only a refactor. If you want to change the log messages do it in a different patch.